components/DEPS

```diff
@@ -38,7 +38,7 @@ specific_include_rules = {
     "+brave/browser",
     "+chrome",
   ],
-  "webcompat_exceptions_browsertest\.cc": [
+  "webcompat_exceptions_unittest\.cc": [
     "+brave/browser",
```

Comment:
I don't think it needs this one anymore

Reply:
There may be some others and possibly deps that can be removed along with the headers that were removed

Reply:
Good catch -- I removed that line. I wasn't able to find other deps to remove.